src/nomad_neb_workflows/schema_packages/neb.py

```diff
@@ -19,7 +23,58 @@
 m_package = SchemaPackage()
 
 
+class NEBWorkflowResults(ArchiveSection):
+    """
+    A section used to define the results of a Nudged Elastic Band (NEB) workflow. This section contains
```

Comment:
I would expand this significantly to give an overview of this automation that you have implemented